docs/resources/Guild_Scheduled_Event.md

```diff
@@ -51,9 +51,10 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
 
 ###### Guild Scheduled Event Entity Metadata
 
-| Field        | Type                | Description                       |
-| ------------ | ------------------- | --------------------------------- |
-| location?    | string              | location of the event             |
+| Field        | Type                | Description                               |
+| ------------ | ------------------- | ----------------------------------------- |
+| speaker_ids? | array of snowflakes | the speakers of the stage channel         |
```

Comment:
oh can we remove speaker_ids, I think this is removed on master. might just need to pull latest and rebase

Reply:
Removed.